gulpfile.js

```diff
@@ -23,6 +23,8 @@ var srcRecipes = recipeSlugs.map(function makePath(name) {
   return './' + name + '/**';
 });
 
+var categoryOrder = ['General Usage', 'Offline', 'Beyond Offline', 'Performance', 'Web Push'];
```

Comment:
Could you add an assertion to check that a recipe belongs to one of these categories?